agent/consul/acl_endpoint_test.go

```diff
@@ -161,7 +161,7 @@ func TestACLEndpoint_TokenRead(t *testing.T) {
 
 	waitForLeaderEstablishment(t, srv)
 
-	acl := ACL{srv: srv}
+	aclEp := ACL{srv: srv}
```

Comment:
This change creates a lot of noise in this file, but I think this is the right thing to do given that the `consul/acl` package was already imported and these `acl` vars where hiding that package in the tests. I really like the disambiguation here.